webapp/src/app/core/IssueCard/issue-card.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, input } from '@angular/core';
+import { PullRequestLabel, PullRequestReview } from '@app/core/modules/openapi';
+import { NgIcon } from '@ng-icons/core';
+import { octCheck, octComment, octFileDiff, octGitPullRequest, octGitPullRequestClosed, octX } from '@ng-icons/octicons';
+import { Dayjs } from 'dayjs';
+import { NgStyle } from '@angular/common';
+
+@Component({
+  selector: 'app-issue-card',
+  templateUrl: './issue-card.component.html',
+  imports: [NgIcon, NgStyle],
+  standalone: true
+})
+export class IssueCardComponent {
+  title = input.required<string>();
+  number = input.required<number>();
+  additions = input.required<number>();
+  deletions = input.required<number>();
+  url = input.required<string>();
+  repositoryName = input.required<string>();
+  reviews = input.required<Array<PullRequestReview>>();
+  createdAt = input.required<Dayjs>();
+  state = input.required<string>();
```

Comment:
You can use the OpenAPI generated type here (make sure to import it):
```suggestion
  state = input.required<PullRequest.StateEnum>();
```